src/main/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1.java

```diff
@@ -2,6 +2,7 @@
 
 import java.util.List;
 
+import edu.hawaii.its.api.type.*;
```

Comment:
It looks like IntelliJ automatically replace the types into a single import *. Remove this * import and import the types individually, thanks.

I will probably create a ticket to fix our codestyle formatter to stop doing that.